docs/challenge-001.md

```diff
@@ -1,9 +1,43 @@
-# Challenge 1: Connect Azure Function to Azure CosmosDB Change Feed
+# Challenge 1: Connect Azure Functions as Change Feed Listeners for Azure CosmosDB Containers
 
 ## Challenge Objective
 
-## Brief Guidance
+**Congratulations!**
+
+You have just completed setting up your development environment and ready to put together the first part of the solution.
+
+As you read in the [solution design document](./solution-design.md), we process two key types of inventory data for a given store - On Hand and Shipments. These are stored in CosmosDB database on Azure, in different containers. As data arrives in these containers, we being our processing.
+
+In this challenge we will begin that processing by building Change Feed Listeners using Azure Functions.
+
+## Guidance
+
+As part of this challenge we are focusing on the following red circled area of the design.
+
+![Focus for Challenge 1](./images/OpenHack_ProcessingFlow_Challenge01Focus.png)
+
+To accomplish this challenge, you will have to connect to your Azure Account and create a new Azure CosmosDB Account and database called - `inventory` using the **SQL API**.
+
+In the CosmosDB database, create three containers - `shipments` (partition key: `/id`) and `onHand` (partition key: `/id`) and `mdc` (partition key: `/id`).
+
+For the change feed listeners, create two Azure Functions with CosmosDB as the trigger. When triggered, the Azure function list out the number of changed document in the change feed and id of the first document.
+
+Insert a document in `onHand` and `shipment` collection using Azure Storage Explorer and validate the Azure Function as been triggered.
```

Comment:
Should we have them use the data generator at this point to create and submit documents of the right shape?

Reply:
We can include that in a story for "moving the data generator" as one of the acceptance criteria to update the documentation .